packages/compass-shell/src/modules/worker-runtime/worker-shell-result.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * TODO: A bunch of things here are copy-pasted from browser-repl, but can be
+ * shared
+ */
+
+import { inspect } from './todo_this_needs_to_be_a_separate_package__inspect';
+
+function serializeProfileResult({ result, count }) {
```

Comment:
Not so much about this PR. But how is it working now?

This one looks like a presentation concerns that (at least at the moment) would go in the React code, along to the other formatting elements.

Why is this not done in `ShowProfileOutput`?

Reply:
It is copy-pasted from `ShowProfileOutput` and maybe works with no issues, I just wasn't sure if there might be any data in `ShowProfileResult` that runtime returns that will not survive serialization, so moved it here. This is definitely something to better examine when doing a final implementation